decoder/pokestop.go

```diff
@@ -193,20 +193,26 @@ func (stop *Pokestop) updatePokestopFromFort(fortData *pogo.PokemonFortProto, ce
 	stop.PowerUpPoints = null.IntFrom(int64(fortData.PowerUpProgressPoints))
 	stop.PowerUpLevel, stop.PowerUpEndTimestamp = calculatePowerUpPoints(fortData)
 
+	// lasModifiedMs is also modified when incident happens
 	lastModifiedTimestamp := fortData.LastModifiedMs / 1000
 	stop.LastModifiedTimestamp = null.IntFrom(lastModifiedTimestamp)
 
 	if len(fortData.ActiveFortModifier) > 0 {
 		lureId := int16(fortData.ActiveFortModifier[0])
 		if lureId >= 501 && lureId <= 510 {
 			lureEnd := lastModifiedTimestamp + LureTime
+			oldLureEnd := stop.LureExpireTimestamp.ValueOrZero()
 			if stop.LureId != lureId {
 				stop.LureExpireTimestamp = null.IntFrom(lureEnd)
 				stop.LureId = lureId
 			} else {
 				now := time.Now().Unix()
-				if now > (lureEnd + 30) { // wait some time after lure end before a restart in case of timing issue
```

Comment:
if the same lure is deployed again (normal -> normal) the lureEnd (lastModfified + 1800) variable is always in the future. therefore we never updated the lure expire timestamp :confused:

Reply:
What doughnut wrote that code! Looks good to me, merge upon successful test